src/utils/hooks/useLoadTemplate.tsx

```diff
@@ -46,14 +46,23 @@ const useLoadTemplate = (props: useLoadTemplateProps) => {
       return
     }
 
-    const { id, code, name, templateSections } = template
+    const {
+      id,
+      code,
+      name,
+      startTitle,
+      startMessage,
```

Comment:
See my back-end comments -- I don't think we need seperate Title and Message fields. Markdown should be able to handle it all as one "Message" field.